bn-apps/memberships-management/membership-service/src/main/kotlin/net/corda/businessnetworks/membership/member/support/BusinessNetworkAwareInitiatedFlow.kt

```diff
@@ -12,10 +12,10 @@ import net.corda.core.identity.Party
  * the initiating party is also a member. Your code (inside onCounterpartyMembershipVerified) will be called only after
  * that check is performed. If the initiating party is not a member an exception is thrown.
  */
-abstract class BusinessNetworkAwareInitiatedFlow<out T>(private val flowSession: FlowSession) : FlowLogic<T>() {
+abstract class BusinessNetworkAwareInitiatedFlow<out T>(protected val session: FlowSession) : FlowLogic<T>() {
```

Comment:
Changing flowSession -> session will mean change in every single flow extending from this. And that's now half of LEIA and Cinsure too. Is it necessary?

Reply:
Reverted back :)